lib/tool_shed/grids/repository_grids.py

```diff
@@ -1529,15 +1526,14 @@ def get_value( self, trans, grid, repository_metadata ):
                                 datatype_tups.append( ( extension, dtype ) )
                         sorted_datatype_tups = sorted( datatype_tups, key=lambda datatype_tup: datatype_tup[ 0 ] )
                         num_datatype_tups = len( sorted_datatype_tups )
-                        for index, datatype_tup in enumerate( sorted_datatype_tups ):
-                            extension = datatype_tup[ 0 ]
-                            dtype = datatype_tup[ 1 ]
+                        datatype_list = []
```

Comment:
This line should be moved at the start of the function, otherwise the return breaks when some `if` is not satisfied.